app/controllers/admin/ticket_scannings_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class TicketScanningsController < Admin::BaseController
+    before_action :authenticate_user!
+    before_action :find_physical_ticket_by_token
+    load_and_authorize_resource through: :physical_ticket, singleton: true, shallow: true
```

Comment:
then may be better to authorize it inside the create action, in a similar way as this is done with events: https://github.com/openSUSE/osem/blob/master/app/controllers/proposals_controller.rb#L38